### DISCUSSION TITLE
Private fields Don't Work in Javascript

### DISCUSSION THREAD

**Discussion by Errorbot1122** [codesandbox/codesandbox-client#5158] [Q&A]
Whenever I try to run a class with a private var it says this:

**`SyntaxError: [FileLocation]: Unexpected character '#' ([Row]:[Col])`**

**Code:**
```
class RandomStuff {
  #LOL = "LOL";
  
  constructor() {
    console.log(this.#LOL)
  }
}
```
and also, Whenever I try to run a class with a private method it says this:

**`Parsing error: This experimental syntax requires enabling the parser plugin: 'classPrivateMethods'
`**

**Code:**
```
class RandomStuff {
  #oof() {
    console.log("oooooooof")
  }
  
  constructor() {
    console.log(this.#LOL)
  }
}
```

What do I do to fix

[Explantion Sandbox](https://codesandbox.io/s/question-private-fields-vxw1j)

**Comment by SaraVieira** [comment]
Hey

The issue here is that we don't support private methods by default you will need to add a babelrc and install the plugins required like so:

https://codesandbox.io/s/question-private-fields-forked-riu3t?file=/src/index.js

Unfortunatly eslint still complains and I will take a look at that but now seems to work

**Reply by Errorbot1122** [reply]
So on my [bigger](https://github.com/ErrorBot1122/Dev-Tools/blob/main/package.json) project, I added     

`"@babel/plugin-proposal-private-methods",` and, `"@babel/plugin-proposal-class-properties"` to the .babelrc file but I still did not work

**Reply by Errorbot1122** [reply]
also, is there a way to access the .eslintrc.* file or an eslintConfig field in a package.json file

**Reply by SaraVieira** [reply]
About your project I do not see any .babelrc, do you have a sandbox you can send me?

Unfortunately so far there is no way to change the eslint options :(

**Reply by Errorbot1122** [reply]
okay i hope thay fix that

**Reply by SaraVieira** [reply]
We are trying to see the best way to fix that
